src/gui/gtk.c

```diff
@@ -1136,7 +1136,7 @@ int dt_gui_gtk_init(dt_gui_gtk_t *gui)
   gtk_menu_item_set_submenu(GTK_MENU_ITEM (view_root_menu), view_menu);
 
   // Help menu
-  GtkWidget *help_root_menu = gtk_menu_item_new_with_label("Help");
+  GtkWidget *help_root_menu = gtk_menu_item_new_with_label(C_("menu", "Help"));
```

Comment:
And no capital letter for Help please.

Reply:
> And no capital letter for Help please.

I'm not sure about this because this is the title of the menu entry.

Reply:
I need to disagree here because the [Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/menus) clearly states that:

> For consistent labels, adopt a condensed style that doesn’t include articles, and use title-style capitalization.

Reply:
Right, this is for the Apple UI, so it is ok as-is. Thanks.